Pod/Classes/LSTracer.m

```diff
@@ -338,30 +338,39 @@ - (void) _refreshImp {
     }
     if (sleepSecs > 0) {
         NSLog(@"LSTracer backing off for %@ seconds", @(sleepSecs));
-        [NSThread sleepForTimeInterval:sleepSecs];
```

Comment:
For future development, it'd be good to update the comment on line 323 to reflect these changes.